appengine-standard-java8/springboot-appengine-standard/pom.xml

```diff
@@ -21,7 +21,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <spring.boot.version>2.2.5.RELEASE</spring.boot.version> <!-- DO NOT UPDATE w/o MANUAL TESTING -->
+    <spring.boot.version>2.2.6.RELEASE</spring.boot.version> <!-- DO NOT UPDATE w/o MANUAL TESTING -->
```

Comment:
@averikitsch or @lesv does this need manual testing? or is the renovate update fine?

Reply:
Not usually.